src/common/src/array/arrow/arrow_impl.rs

```diff
@@ -1190,11 +1190,24 @@ impl FromIntoArrow for Interval {
     type ArrowType = ArrowIntervalType;
 
     fn from_arrow(value: Self::ArrowType) -> Self {
-        <ArrowIntervalType as crate::array::arrow::ArrowIntervalTypeTrait>::to_interval(value)
+        // XXX: the arrow-rs decoding is incorrect
+        // let (months, days, ns) = arrow_array::types::IntervalMonthDayNanoType::to_parts(value);
```

Comment:
@xxchan 
Is this comment still valid?
#18608

Reply:
I'm not sure. This seem to exist since the beginning https://github.com/risingwavelabs/risingwave/pull/10399

Reply:
Thank you and just removed it